wdl2cwl/main.py

```diff
@@ -28,18 +28,23 @@
 # eval is not on the official list of reserved words, but it is a built-in function
 
 
-def convert(doc: str) -> cwl.CommandLineTool:
+def convert(doc: str) -> Dict[str, List[str]]:
```

Comment:
Isn't that `Dict[str, Any]`, as that's the type returned by `miniwdl` functions used within this function?